osh/hub/scripts/osh-xmlrpc-client.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
```

Comment:
Why is this needed?  For example `osh/hub/manage.py` still contains `#!/usr/bin/env python3`.

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#_shebang_lines

Reply:
If you remove the scripts in `%install`, you do not need these changes.  `manage.py` should not removed though.